CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
   `keyAuthorization` field but will temporarily retry the request with the
   field included if a `malformed` error is received. This fallback will be
   removed in version 0.34.0.
+* cli is now a package split in submodules instead of a whole module.
```

Comment:
I couldn't agree more with what you wrote ! We stopped because the project was over, but we knew we could again split it more. However, having dozens of submodules can also make it a bit hard to understand what cli does in the end when looking at __init.py. I don't think it's happening here, but it should be as atomic as possible and also keeping groups of methods that implement a specific feature.

Again, don't forget to add tests (even though it can be painful to write!), the purpose of this refactoring was to make cli easier to read and easier to test :) But as I said, it could be in a future PR, it might be too much for a single PR.

Reply:
Indeed, tests should be refactored in a later PR. For now, the coverage is the same, so no rush on it.

For this PR, will you have the time to apply the suggestions?